client/src/components/Prompt.svelte

```diff
@@ -76,7 +82,10 @@
     <Textfield input$id="prompt-answer" bind:value={answer} bind:dirty invalid={dirty && error != undefined} required>
       <HelperText validationMsg slot="helper">{error ?? ""}</HelperText>
     </Textfield>
-    <Button type="submit" disabled={error != undefined}><Label>Done</Label></Button>
+    <Button type="submit" disabled={error != undefined} on:click={showAnswer}><Label>Done</Label></Button>
+    {#if showUserAnswer}
```

Comment:
why not just do `displayAnswer != ""` or even make displayAnswer start as undefined and display if it is not undefined